.github/workflows/ci-osx.yml

```diff
@@ -78,3 +78,25 @@ jobs:
         # To find available versions, add the following as a step above:
         #  - run: ls -l /Applications
         DEVELOPER_DIR: /Applications/Xcode_11.3.1.app/Contents/Developer
+
+    - name: Send failure mail to dynamorio-devs
```

Comment:
I don't really like this duplication... but even if we create our own custom action with the common stuff, we'll remove only 7/21 of these lines (will need to keep the ones that access env vars, and the ones required to invoke the new action).

Reply:
We already have a lot of duplication among our 6 workflow files and it's sort of a pain to add something common b/c you have to edit 6 files...but OTOH hopefully we'll not have to touch them again for years and it's prob not worth a lot of effort to share.